resolvers/webpack/test/files/webpack.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     { 'jquery': 'jQuery' },
     'bootstrap',
     function (context, request, callback) {
-      if (request === 'underscore') {
+      if (request === 'underscore' && context.indexOf('/') !== -1) {
```

Comment:
It is just to force an error when `context` is `null` since `null` has no `indexOf` method.  Whatever produces an error when `context` is null would do just as well, just to make sure the fix is not reversed later on.